fixtures/app/extensions/customer-accounts-ui/locales/en.default.json

```diff
@@ -0,0 +1,3 @@
+{
+  "welcome": "Welcome to the customer-accounts-ui extension!"
+}
```

Comment:
Is it relevant to add the locales to the template as well, the same way it is done in checkout-ui extension? thus a sample of localization will be given to the developer on extension creation.